server/src/main/java/io/crate/execution/engine/collect/DocValuesAggregates.java

```diff
@@ -160,7 +161,10 @@ public static List<DocValueAggregator> createAggregators(Functions functions,
                         // to the normal aggregation implementation
                         return null;
                     }
-                    aggregationReferences.add(reference);
+                    // Ignore source look-ups because the field does not exist in doc-values and will therefore
+                    // not work with aggregations
+                    var referenceWithoutSourceLookUp = (Reference) DocReferences.inverseSourceLookup(reference);
+                    aggregationReferences.add(referenceWithoutSourceLookUp);
```

Comment:
I think switching over to regular references would be worthwhile for aggregations even outside of the `DocValuesAggregates`.

Could be done in the planner, similiar to https://github.com/crate/crate/pull/11498/files#diff-9ca8b12f552725efbcc105a2530e9e6d74ae7f91dcb545c95c30132219b59f95 

(And maybe also check the performance impact, https://github.com/crate/crate/pull/11498 had quite an improvement9

Reply:
Thanks for the feedback.